apps/cowswap-frontend/src/modules/account/containers/Transaction/StatusDetails.tsx

```diff
@@ -130,6 +131,8 @@ export function StatusDetails(props: StatusDetailsProps) {
           <CancelButton onClick={showCancellationModal} />
         </StatusLabelBelow>
       )}
+      {/* TODO: Probably not the right component, just placeholder for now */}
+      {showProgressBar && <CancelButton onClick={showProgressBar}>Show progress</CancelButton>}
```

Comment:
Very ugly, I know. I hope @fairlighteth will have a better idea where to put and how to display this.
![image](https://github.com/user-attachments/assets/0ee41b47-5a28-40b4-985a-0dded0fc11f5)

Reply:
Will review